scripts/postcss/css-url-to-variables.js

```diff
@@ -81,7 +81,8 @@ module.exports = (opts = {}) => {
             const urlVariables = new Map();
             const counter = createCounter();
             root.walkDecls(decl => findAndReplaceUrl(decl, urlVariables, counter));
-            if (urlVariables.size) {
+            const cssFileLocation = root.source.input.from;
+            if (urlVariables.size && !cssFileLocation.includes("type=runtime")) {
```

Comment:
For runtime themes, we shouldn't add the extracted icon variables to the `:root` section because we'll be adding them at runtime.